packages/lab/src/stepped-tracker/utils.ts

```diff
@@ -1,22 +1,65 @@
-import type { StepRecord, StepStage } from "./Step.types";
+import type { StepRecord, StepStage, StepStatus } from "./Step.types";
 import type { StepReducerOptions, StepReducerState } from "./stepReducer.types";
 
-export function assignSteps(
+export function assignStepsStage(
```

Comment:
Mention rename? This is public and users can use .?

Reply:
Nope, this is internal  use only. Users have to go through `useStepReducer` to utilize that functionality.